language/move-lang/src/errors/new.rs

```diff
@@ -193,14 +193,26 @@ impl Diagnostic {
     pub(crate) fn add_secondary_label(&mut self, (loc, msg): (Loc, impl ToString)) {
         self.secondary_labels.push((loc, msg.to_string()))
     }
+
+    pub(crate) fn secondary_labels_len(&self) -> usize {
+        self.secondary_labels.len()
+    }
```

Comment:
So this function is only used for an `assert!` in `language/move-lang/src/cfgir/borrows/state.rs`? Since `self.secondary_labels` is also `pub(crate)`, could that file use `self.secondary_labels.len()`, and this function be deleted? Or would you prefer users not access `secondary_labels` directly?

Reply:
I wouldn't want that really. The only API outside of this file should be push/extend/len. I don't want to expose that it is a Vec

Reply:
That makes sense to me. It seems to me that `secondary_labels` itself is `pub(crate)` though, is that just part of the migration?

Reply:
Indeed, yeah. It's needed for the old/new items to share some code for translating to the move-prover's error format